### PR TITLE
52 Update notary endpoints to provide user info and add bio to notary table

### LINKED ISSUE #52: Update model to remove is_notary
Update model to remove `is_notary` and instead use the select_related method.

### PR DESCRIPTION
**Pull request by davidlopez**
## Type of change
- [x] :page_with_curl: This change requires a documentation update

## Summary of changes
- Update user and notary models to add `bio` and remove `is_notary`
- Update notary endpoints to provide notary/user info

## Checklist:
- [x] :white_check_mark: I have performed a self-review of my own code
- [x] :eight_spoked_asterisk: I have commented my code, particularly in hard-to-understand areas
- [ ] :page_facing_up: I have made corresponding changes to the documentation
- [x] :no_entry_sign: My changes generate no new warnings
- [x] :100: New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by BabsLabs — APPROVED** [review]
This looks great. I appreciate you taking the time to show me how you used the select_related method.

File: notary_now_api/migrations/0003_notary_bio.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('notary_now_api', '0002_auto_20200219_0101'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='notary',
+            name='bio',
```
Comment (lrs8810) [inline notary_now_api/migrations/0003_notary_bio.py:15]: Good catch!

File: notary_now_api/models.py [hunk 2]
```diff
@@ -80,6 +79,7 @@ class Notary(models.Model):
     user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, related_name='notary_user')
     radius = models.PositiveIntegerField()
     active = models.BooleanField(default=True)
+    bio = models.TextField(max_length=5000)
```
Comment (lrs8810) [inline notary_now_api/models.py:82]: Good catch!

File: notary_now_api/views.py [hunk 3]
```diff
@@ -1,26 +1,52 @@
 from django.shortcuts import render, get_object_or_404
-from django.http import JsonResponse
+from django.http import JsonResponse, HttpResponse
 from .models import User, Notary
 
 def notary_users_list(request):
-    users = User.objects.filter(is_notary='True')
+    notaries = Notary.objects.select_related('user')
```
Comment (lrs8810) [inline notary_now_api/views.py:6]: Great use of `select_related`!

File: notary_now_api/views.py [hunk 4]
```diff
@@ -1,26 +1,52 @@
...
 
-    data = {"results": list(users.values("id", "first_name", "last_name", "profile_photo", "zip_code"))}
-    
-    return JsonResponse(data)
+    notary_data = []
 
-def notary_detail(request, pk):
-    user = get_object_or_404(User, pk=pk)
+    for notary in notaries:
```
Comment (lrs8810) [inline notary_now_api/views.py:10]: Nice formatting!

File: notary_now_api/views.py [hunk 5]
```diff
@@ -1,26 +1,52 @@
...
+        notary = notary.select_related('user')[0]
+        data = {
+            "id": notary.user.id,
+            "first_name": notary.user.first_name,
+            "last_name": notary.user.last_name,
+            "email": notary.user.email,
+            "profile_photo": notary.user.profile_photo,
+            "zip_code": notary.user.zip_code,
+            "notary_values": {
+                "commission_date": notary.commission_date,
```
Comment (lrs8810) [inline notary_now_api/views.py:42]: Nice formatting!

**Review by lrs8810 — APPROVED** [review]
Great job!

**Pull request merged**